site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+        "authenticatorGetInfo": {
+          "versions": [
+            "FIDO_2_0"
+          ],
+          "aaguid": "08987058cadc4b81b6e130de50dcbe96",
+          "options": {
+            "plat": true,
+            "rk": true,
+            "up": true
+          },
+          "maxCredentialCountInList": 100,
+          "maxCredentialIdLength": 32,
+          "transports": [
+            "internal"
+          ],
+          "algorithms": [
+            {
+              "type": "public-key",
+              "alg": -257
+            }
+          ],
+          "minPINLength": 4,
+          "firmwareVersion": 19042
+        }
+      },
+      "statusReports": [
+        {
+          "status": "FIDO_CERTIFIED_L1",
+          "effectiveDate": "2020-08-05",
+          "certificationDescriptor": "Windows Hello Hardware Authenticator",
+          "certificateNumber": "FIDO20020190418002",
+          "certificationPolicyVersion": "1.3.6",
+          "certificationRequirementsVersion": "1.1.0"
+        }
+      ],
+      "timeOfLastStatusChange": "2020-08-05"
+    }
+```
+
+There is a lot of structured metadata that you can use if desired.
+
+## User Presence vs User Verification
+
+Authenticators can be also used as a second factor of authentication, when paired with a username and password or other factor. An authentication ceremony gives you assurances about the user having access to the authenticator. If you are using an authenticator in this way, you want to distinguish between user presence versus user verification. 
+
+User presence is when someone is there behind the authenticator who gave permission, but you don't know who. All it proves is that you have access to the authenticator. An example of this is when you press a button on a Yubikey.
+
+User verification, on the other hand, is when the user provides proof that they are who they say they are. The authenticator could require a pin, known only to the user. It could be a Face ID or Touch ID challenge. In any case, verification allows you to tie a specific user to the authentication ceremony.
```

Comment:
```suggestion
User verification, on the other hand, is when the user provides proof that they are who they say they are. The authenticator could require a PIN, known only to the user. It could be a Face ID or Touch ID challenge. In any case, verification allows you to tie a specific user to the authentication ceremony.
```